src/ros_filter.cpp

```diff
@@ -1436,7 +1445,7 @@ void RosFilter::loadParams()
       deceleration_gains);
 
     control_sub_ = node_->create_subscription<geometry_msgs::msg::Twist>(
-      "cmd_vel",
+      "cmd_vel", 10, 
```

Comment:
queue size

Reply:
This parameter would be outside the scope of odom, twist, pose and imu. Do we want to declare a separate parameter for this one?

Reply:
Oh sorry this one is fine. I went a little trigger happy when I saw the number `10` :wink: